evan_did_method_spec.md

```diff
@@ -95,18 +97,25 @@ It is also possible to add and remove service endpoints that are to be associate
 You must be either the owner or the controller of the identity to apply modifications and updates can only be performed by SSIs.
 
 ```javascript
-{
-    '@context': 'https://w3id.org/did/v1',
-    id: 'did:evan:0x071be9398b832d711b2e6278eba372ec968d8073e33a9cac9289c3ed06fc0701',
-    controller: 'did:evan:0xb9A7884DEf4F8E503C50068dCB03d2393cBDFA33',
-    authentication: [
-        'did:evan:0xb9A7884DEf4F8E503C50068dCB03d2393cBDFA33#key-1'
-    ],
-    service: [{
-        "id": "did:evan:0x071be9398b832d711b2e6278eba372ec968d8073e33a9cac9289c3ed06fc0701#messaging",
-        "type": "MessagingService",
-        "serviceEndpoint": "https://message.me.example.com/"
-    }]
+{ '@context': 'https://w3id.org/did/v1',
+  id:
+   'did:evan:testcore:0x0d87204c3957d73b68ae28d0af961d3c72403906',
+  publicKey:
+   [ { id:
+        'did:evan:testcore:0x0d87204c3957d73b68ae28d0af961d3c72403906#key-1',
+       type: 'Secp256k1VerificationKey2018',
+       controller:
+        'did:evan:testcore:0x0d87204c3957d73b68ae28d0af961d3c72403906',
+       ethereumAddress: '0xcd5e1dbb5552c2baa1943e6b5f66d22107e9c05c' } ],
+  authentication:
+   [ 'did:evan:testcore:0x0d87204c3957d73b68ae28d0af961d3c72403906#key-1' ],
+  created: '2020-03-19T09:56:33.808Z',
+  updated: '2020-03-19T09:56:34.563Z',
+  service:
+   [ { id:
+        'did:evan:testcore:0x0d87204c3957d73b68ae28d0af961d3c72403906#randomService',
+       type: 'randomService-575000253',
+       serviceEndpoint: 'https://openid.example.com/575000253' } ] 
 }
```

Comment:
Could use a proof property

Just use the document from [here](https://testcore.evan.network/did/did:evan:testcore:0x0d87204c3957d73b68ae28d0af961d3c72403906)